challenge.rb

```diff
@@ -57,7 +76,17 @@ def ac_monitor current_temp, ac_working, desired_temp
 # - return "fizzbuzz" if `num` is evenly divisible by both 3 and 5
 # - return the value of `num` if `num` isn't divisible by either 3 or 5
 
-# Replace this comment with your code.
+def fizz_buzz_calculator num
+  if num % 3 == 0 && num % 5 == 0
+    return "fizzbuzz"
+  elsif num % 3 == 0
+    return "fizz"
+  elsif num % 5 == 0
+    return "buzz"
+  else
+    return num
+  end
+end
```

Comment:
Great! One small nit - the `return` statements aren't needed here, because a Ruby method or block will return the value of the last evaluated expression by default.